src/components/QuranReader/TranslationView/TranslationText/TranslationAndReference.tsx

```
@@ -0,0 +1,69 @@
+  text,
+  fontClass,
+  direction,
+  onTextClicked,
+  shouldShowReference = false,
+  chapterName,
+  reference,
+  lang,
+}) => {
+  return (
+    <div
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)